assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -332,6 +332,124 @@ footer[role="contentinfo"] a {
 
 }
 
+// Alternative form layout 
+.gu-content--contribution-form-two {
+  header[role="banner"] {
+
+    @include mq($from: tablet) {
+      height: 120px;
```

Comment:
do we need this height? could the header have padding here instead and let the logo determine the height? that way we have 1 less magic number